From 86f0964dfa4ba879564f32cc7ccfc9c84dffb7f5 Mon Sep 17 00:00:00 2001
From: Jonas Haag <jonas@lophus.org>
Date: Sun, 9 Jul 2023 20:44:23 +0200
Subject: [PATCH 1/2] Add missing includes

---
 dltest/dltest.c | 1 +
 test/odbctap.h  | 1 +
 2 files changed, 2 insertions(+)

diff --git a/dltest/dltest.c b/dltest/dltest.c
index d3b9360..3fb354b 100644
--- a/dltest/dltest.c
+++ b/dltest/dltest.c
@@ -28,6 +28,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 
 #ifdef WIN32
     #include <windows.h>
diff --git a/test/odbctap.h b/test/odbctap.h
index 5bae044..e8b2a43 100644
--- a/test/odbctap.h
+++ b/test/odbctap.h
@@ -55,6 +55,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <wchar.h>
 
 #ifdef _WIN32
 /* Driver does that - applications should do that too, to have same size of structures */
-- 
2.41.0

