From 304a6d07555a97b3372e14e9bca9971832605d12 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Wed, 21 Feb 2024 20:38:24 +0100
Subject: [PATCH] Add missing includes

---
 test/odbctap.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/odbctap.h b/test/odbctap.h
index 54c117a..6f3c31c 100644
--- a/test/odbctap.h
+++ b/test/odbctap.h
@@ -55,6 +55,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <wchar.h>
 
 #ifdef _WIN32
 /* Driver does that - applications should do that too, to have same size of structures */
